apps/server/src/controllers/create-tournament.controller.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  PROTO_CREATE_LOBBY_ACCEPT,
+  PROTO_CREATE_LOBBY_REQUEST,
+} from '@razor/constants';
+import {
+  InitialClientData,
+  InitialServerData,
+  Snapshot,
+  socketId,
+} from '@razor/models';
+import { store } from '@razor/store';
+
+import { PubSubEvents } from '../models';
+import { ContextOutput, Logger, pubsub } from '../services';
+import { tokenPlayerMap } from '../stores';
+
+interface JoinLobbyRequestArgs {
+  socketId: socketId;
+  data: InitialClientData;
+  context: ContextOutput;
+}
+
+const logger = new Logger('create-tournament.controller');
+
+pubsub.subscribe(
+  PROTO_CREATE_LOBBY_REQUEST,
+  ({ socketId, data, context }: JoinLobbyRequestArgs) => {
```

Comment:
shall we extract this to a separate function instead of inlining it

Reply:
Completed in c762ae43461576710016979f76af2ddf08ed2dba #81